### PR TITLE
implemented parser, which includes ld info into the rede json schema

### PR DESCRIPTION
**Pull request by stefan-sev**

### CODE REVIEW

**Comment by jbedo** [comment]
That shell script is very scary:
- there's no checking of errors;
- rm's in _current_ directory with wildcards!

You need to be much more careful with creating temporary files and cleaning them up.  Create a temporary directory (say /tmp/$$), put your temporary files there, and remove the directory to clean up.  Check for errors, catch them, clean up your temporary files.  You're also missing the #! line at the top of the file.

File: scripts/ld_generator/add_ld_data.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Author: Stefan Sevelda
```
Comment (jbedo) [inline scripts/ld_generator/add_ld_data.sh:3]: #!/bin/sh

File: scripts/ld_generator/add_ld_data.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            --form includeQuerySnp=on \
+            --form arrayFilter=query \
+            --form columnList[]=DP \
+            --form columnList[]=GP \
+            --form columnList[]=AM \
+            http://www.broadinstitute.org/mpg/snap/ldsearch.php > SNAPResults$i.txt
+    done
+
+# remove all the opensnp..txt files
+rm opensnp*.txt
```
Comment (jbedo) [inline scripts/ld_generator/add_ld_data.sh:57]: !

File: scripts/ld_generator/add_ld_data.sh [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+# parses all the information back to the json schema
+python parse_back_to_json.py $1 $2
+
+# removes all the txt files of snap
+rm SNAPR*.txt
```
Comment (jbedo) [inline scripts/ld_generator/add_ld_data.sh:63]: !

**Comment by chengsoonong** [comment]
Also with #!/bin/sh, once you chmod u+x, you can execute it with ./add_ld_data.sh ...

**Pull request closed without merging**